requirements-freqai.txt

```diff
@@ -0,0 +1,8 @@
+# Include all requirements to run the bot.
+-r requirements.txt
+
+# Required for freqai
+scikit-learn==1.0.2
+scikit-optimize==0.9.0
+joblib==1.1.0
+catboost==1.0.4
```

Comment:
It seems like installation of catboost on ARM is not possible as they don't provide ARM wheels via pypi.

are there instructions / workarounds for that available (if yes, they should be in the docs) ...

Reply:
Noted, I will make sure to make LightGBM the default prediction model, and add some info about this in the docs ;)